Marlin/src/pins/pins_BIGTREE_SKR_PRO_V1.1.h

```diff
@@ -0,0 +1,235 @@
+#endif
+
+#define EEPROM_EMULATED_WITH_SRAM
+
+#if HOTENDS > 3 || E_STEPPERS > 3
+  #error "BIGTREE SKR Pro V1.1 supports up to 3 hotends / E-steppers."
+#endif
+
+//
+// Servos
+//
+#define SERVO0_PIN        PA1
+
+//
+// Limit Switches
+//
+#define X_MIN_PIN          PB10
+#define X_MAX_PIN          PE15
+#define Y_MIN_PIN          PE12
+#define Y_MAX_PIN          PE10
+#define Z_MIN_PIN          PG8
+#define Z_MAX_PIN          PG5
+
+//
+// Z Probe must be this pins
+//
+#ifndef Z_MIN_PROBE_PIN
+  #define Z_MIN_PROBE_PIN  PA2
+#endif
+
+//
+// Steppers
+//
+#define X_STEP_PIN         PE9
+#define X_DIR_PIN          PF1
+#define X_ENABLE_PIN       PF2
+#ifndef X_CS_PIN
+  #define X_CS_PIN         PA15
+#endif
+
+#define Y_STEP_PIN         PE11
+#define Y_DIR_PIN          PE8
+#define Y_ENABLE_PIN       PD7
+ #ifndef Y_CS_PIN
+  #define Y_CS_PIN         PB8
+#endif
+
+#define Z_STEP_PIN         PE13
+#define Z_DIR_PIN          PC2
+#define Z_ENABLE_PIN       PC0
+#ifndef Z_CS_PIN
+  #define Z_CS_PIN         PB9
+#endif
+
+#define E0_STEP_PIN        PE14
+#define E0_DIR_PIN         PA0
+#define E0_ENABLE_PIN      PC3
+#ifndef E0_CS_PIN
+  #define E0_CS_PIN        PB3
+#endif
+
+#define E1_STEP_PIN        PD15
+#define E1_DIR_PIN         PE7
+#define E1_ENABLE_PIN      PA3
+#ifndef E1_CS_PIN
+  #define E1_CS_PIN        PG15
+#endif
+
+#define E2_STEP_PIN        PD13
+#define E2_DIR_PIN         PG9
+#define E2_ENABLE_PIN      PF0
+#ifndef E2_CS_PIN
+  #define E2_CS_PIN        PG12
+#endif
+
+//
+// Software SPI pins for TMC2130 stepper drivers
+//
+#if ENABLED(TMC_USE_SW_SPI)
+  #define TMC_SW_MOSI      PC12
+  #define TMC_SW_MISO      PC11
+  #define TMC_SW_SCK       PC10
+#endif
+
+#if HAS_DRIVER(TMC2208)
+  /**
+   * TMC2208 stepper drivers
+   *
+   * Hardware serial communication ports.
+   * If undefined software serial is used according to the pins below
+   */
+  //#define X_HARDWARE_SERIAL  Serial
+  //#define X2_HARDWARE_SERIAL Serial1
+  //#define Y_HARDWARE_SERIAL  Serial1
+  //#define Y2_HARDWARE_SERIAL Serial1
+  //#define Z_HARDWARE_SERIAL  Serial1
+  //#define Z2_HARDWARE_SERIAL Serial1
+  //#define E0_HARDWARE_SERIAL Serial1
+  //#define E1_HARDWARE_SERIAL Serial1
+  //#define E2_HARDWARE_SERIAL Serial1
+  //#define E3_HARDWARE_SERIAL Serial1
+  //#define E4_HARDWARE_SERIAL Serial1
+
+  //
+  // Software serial
+  //   
+  #define X_SERIAL_TX_PIN  PC13
+  #define X_SERIAL_RX_PIN  PE4
+
+  #define Y_SERIAL_TX_PIN  PE3
+  #define Y_SERIAL_RX_PIN  PE2
+
+  #define Z_SERIAL_TX_PIN  PE0
+  #define Z_SERIAL_RX_PIN  PE1
+
+  #define E0_SERIAL_TX_PIN PD4
+  #define E0_SERIAL_RX_PIN PD2
+
+  #define E1_SERIAL_TX_PIN PD0
+  #define E1_SERIAL_RX_PIN PD1
+
+  #define Z2_SERIAL_TX_PIN PD6
+  #define Z2_SERIAL_RX_PIN PD5
+#endif
+
+//
+// Temperature Sensors
+//
+#define TEMP_0_PIN         PF3  // T0
+#define TEMP_1_PIN         PF4  // T1
+#define TEMP_2_PIN         PF5  // T2
+#define TEMP_BED_PIN       PF6  // TB
+
```

Comment:
Can you confirm these are the correct pins? They don't match the documentation & there's at least one report in the BigTreeTech Facebook group with the correct pins being:

```c++
#define TEMP_BED_PIN       PF3  // T0 <=> TB
#define TEMP_0_PIN         PF4  // T1
#define TEMP_1_PIN         PF5  // T2
#define TEMP_2_PIN         PF6  // T3
```

I'm still waiting on my board to arrive, so I can't confirm.